src/components/Image/Image.js

```diff
@@ -18,12 +18,12 @@ export default class Image extends Component {
     }
   }
 
-  componentWillReceiveProps (nextProps) {
-    if (this.props.src !== nextProps.src) {
-      this.setState({
+  static getDerivedStateFromProps(nextProps) {
+    if (this.props.src !== nextProps.src){
```

Comment:
Since `getDerivedStateFromProps` is static, you can't use `this` here. The current `src` would need to be moved into the state and then you can use `currentState` (the second argument of `getDerivedStateFromProps` to check if it changed.

Reply:
oh right, thanks for the tips